Wikipedia/Code/ReadingThemesControlsPresenter.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  ReadingThemesControlsPresenter.swift
+//  Wikipedia
+//
+//  Created by Toni Sevener on 2/27/19.
+//  Copyright © 2019 Wikimedia Foundation. All rights reserved.
+//
+
+import Foundation
+
+protocol ReadingThemesControlsPresenterProtocol: WMFReadingThemesControlsViewControllerDelegate, UIPopoverPresentationControllerDelegate {
+    var readingThemesControlsViewController: ReadingThemesControlsViewController { get } //lazy loaded
+    var wkWebView: WKWebView { get }
+    var readingThemesControlsToolbarItem: UIBarButtonItem { get }
+    var passthroughNavBar: Bool { get }
+}
+
+extension ReadingThemesControlsPresenterProtocol {
+    
+    var fontSizeMultipliers: [Int] {
+
+        return [WMFFontSizeMultiplier.extraSmall.rawValue,
```

Comment:
Do I need to keep the FBTweakValues from the old Obj-C article code?

Reply:
Nope, we haven't been using `FBTweaks`